### ISSUE TITLE
Network callbacks give error when not existing

### ISSUE THREAD

**Issue by NiLSPACE** [mc-server/MCServer#2208]
When using the network API you can get errors like `[18:46:43] cTCPLink OnRemoteClosed() callback failed in plugin MC-Web-Server.` because the callback is missing. Is this intended?

**Comment by madmaxoft** [comment]
It was done on purpose, but thinking back, I think the purpose is gone. It might be a good idea to get rid of the error messages.

**Comment by NiLSPACE** [comment]
Yeah, it doesn't make sense to have those errors for a missing callback anyway.
